contribuer/public/admin/institution.js

```diff
@@ -50,19 +50,22 @@ class InstitutionControl extends Relation.control {
   }
 
   componentDidMount() {
-    const defaultValue = this.props.field.get("filter").get("default", "*")
-    this.setState({
-      filter: defaultValue,
-      categories: this.props.field
-        .get("filter")
-        .get("fields")
-        .map((option) => {
+    const { query, forID } = this.props
+    query(forID, "institution_types", ["name"], "").then(({ payload }) => {
+      const hits = payload.hits || []
+      const institutionTypes = [
+        { value: "*", label: "Tous" },
+        ...hits.map(hit => {
```

Comment:
En local, je n'ai pas réussi à avoir autre chose qu'une liste avec un seul élément ["Tous"]. Compliqué de valider en l'état...

Soit en revert cette partie là pour garder la partie simple soit on fait une nouvelle itération sur ce bloc.

Reply:
J'ai testé à nouveau en local, ça fonctionne désormais. Désolé pour le faux négatif de la semaine dernière.